login/profile.go

```diff
@@ -136,46 +156,150 @@ func (userProfileClient *KeycloakUserProfileClient) Create(ctx context.Context,
 		// Observed this error code when trying to create user
 		// with a token belonging to a different realm.
 		if resp.StatusCode == 403 {
-			return nil, errors.NewUnauthorizedError(bodyString)
+			return nil, false, errors.NewUnauthorizedError(bodyString)
 		}
 
-		// Observed this error code when trying to create user with an existing username.
-		if resp.StatusCode == 409 {
-			// This isn't version conflict really,
-			// but helps us generate the final response code.
-			return nil, errors.NewVersionConflictError(fmt.Sprintf("user with username %s / email %s already exists", *keycloakUserRequest.Username, *keycloakUserRequest.Email))
-		}
-
-		return nil, errors.NewInternalError(ctx, errs.Errorf("received a non-200 response %s while creating keycloak user :  %s", resp.Status, keycloakAdminUserAPIURL))
+		return nil, true, errors.NewInternalError(ctx, errs.Errorf("received a non-200 response %s while creating keycloak user :  %s", resp.Status, keycloakAdminUserAPIURL))
 	}
-	log.Info(ctx, map[string]interface{}{
-		"response_status":           resp.Status,
-		"response_body":             bodyString,
-		"keycloak_user_profile_url": keycloakAdminUserAPIURL,
-	}, "Successfully create Keycloak user")
 
 	createdUserURL, err := resp.Location()
 	if err != nil {
 		log.Error(ctx, map[string]interface{}{
 			"keycloak_user_url": keycloakAdminUserAPIURL,
 			"err":               err,
 		}, "Unable to create Keycloak user")
-		return nil, errors.NewInternalError(ctx, err)
+		return nil, false, errors.NewInternalError(ctx, err)
 	}
 	if createdUserURL == nil {
 		log.Error(ctx, map[string]interface{}{
 			"keycloak_user_url": keycloakAdminUserAPIURL,
 		}, "Unable to create Keycloak user")
-		return nil, errors.NewInternalError(ctx, errs.Errorf("user creation in keycloak might have failed."))
+		return nil, false, errors.NewInternalError(ctx, errs.Errorf("user creation in keycloak might have failed."))
 	}
 
 	createdUserURLString := createdUserURL.String()
 	log.Info(ctx, map[string]interface{}{
 		"keycloak_user_url": keycloakAdminUserAPIURL,
 		"user_url":          createdUserURLString,
-	}, "Successfully created Keycloak user user")
+	}, "Successfully created Keycloak user")
+
+	return &createdUserURLString, false, nil
```

Comment:
this should be `true` , i'm updating your branch. have the change locally.

Reply:
Oh, right. I added "true" to line 162 instead for some reason :)